WordPress/twentytwentyfive
# Add footers

**Description**
This PR adds footer patterns.
Closes https://github.com/WordPress/twentytwentyfive/issues/38
**Testing Instructions**

Since the font sizes and font family is not in yet, the main thing to test in this PR is that:
- The blocks are the same as [in the designs](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3056-190&t=glA97sx4WeWtz3Ti-4).
- The spacing to the top, bottom, left and right of the footer content, as well as the spacing between the "main" footer and the credits (Designed with...) matches the designs in Figma closely.

Apply the PR.
Confirm that the default footer is used and displays correctly in the editor and front.
Test with different screen widths.
Confirm that the two navigation blocks in the default header has aria-labels. You can do that by viewing the source.

Replace the content in the template part with the other footer patterns from the theme, and test those as well.

(-You will see that the yellow footer is not yellow, the black footer is not black: I added a comment about that)
